python/tvm/relay/frontend/tensorflow.py

```diff
@@ -626,8 +626,14 @@ def _impl(inputs, attr, params):
 
 def _slice():
     def _impl(inputs, attr, params):
-        begin = _get_list_param(params, inputs[1])
-        size = _get_list_param(params, inputs[2])
+        try:
+            begin = _get_list_param(params, inputs[1])
+        except:
+            begin = _infer_value_simulated(inputs[1], params).asnumpy()[0]
```

Comment:
it might be `begin = _infer_value(inputs[1], params).asnumpy().tolist()`.

Reply:
I think `_infer_value_simulated` is better here. `_infer_value` assumes that all of the inputs leading up to `inputs[1]` are constant, `_infer_value_simulated` does not.

Reply:
I'm not sure whether ```_infer_value_simulated``` should be used here. size parameter can be a data dependent expression. If we want to infer the correct value, probably we should enforce all variables in params. For symbolic begin/size slice, it will be covered in https://github.com/apache/incubator-tvm/pull/4312.

Reply:
`_infer_value_simulated ` is not able to handle data dependent cases. Let's use `_infer_value` here, `_infer_value` doesn't help, then we have to use dynamic strided_slice in pr 4312. @lsy643  let us know if `_infer_value` doesn't work for you

Reply:
`_infer_value` works for me. It seems `_infer_value_simulated ` use `_infer_value ` inside, so why prefer one over another?

Reply:
```_infer_value_simulated``` will create dummy data if some variables don't appear in params. In data dependent case, this will cause problem.